examples/lighting-app/telink/include/AppTask.h

```diff
@@ -88,6 +89,9 @@ class AppTask
     static void ThreadProvisioningHandler(const chip::DeviceLayer::ChipDeviceEvent * event, intptr_t arg);
 
     static AppTask sAppTask;
+    PWMDevice mBluePwmLed;
+    PWMDevice mGreenPwmLed;
```

Comment:
should these also be around a `#if USE_RGB_PWM` since the AppTask.cpp seem to set the static structs like that?

Reply:
Fixed